crates/daphne/src/pine/flp.rs

```diff
@@ -56,7 +58,22 @@ use super::{chunk_count, f64_to_field, field_to_f64, PineConfig, USAGE_WR_JOINT_
 /// encoded measurement.
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct PineType<F> {
-    pub(crate) cfg: PineConfig<F>,
+    // PINE parameters
+    pub(crate) dimension: usize,
+    pub(crate) frac_bits: usize,
+    pub(crate) chunk_len: usize,
+    pub(crate) num_wr_tests: usize,
+    pub(crate) num_wr_successes: usize,
+
+    // FLP parameters
+    pub(crate) sq_norm_bound: F,
```

Comment:
is it possible to store the bounds as integers and internally convert them to field elements?
Thus, the 'PineType' struct is not attached to any field in particular.

Reply:
Well, the type of interger would be `F::Integer`: for Field64 this is u64; for Field128 this would be u128. So we'd still have a generic parameter no matter what unfortunately.

Another consideration here: we use this in multiple parts of the code. If we stored it as an integer, we'd have to convert to a field element on the fly, which has some cost.